examples/with-storybook/src/mocks/browser.js

```diff
@@ -0,0 +1,9 @@
+import { setupWorker } from 'msw'
+import { handlers } from './handlers'
+
+console.log('setupWorker', typeof window, typeof global.process)
+
+// Export the worker instance, so we can await the activation on Storybook's runtime.
+// You can use this reference to start the worker for local development as well.
+export const worker =
+  typeof global.process === 'undefined' && setupWorker(...handlers)
```

Comment:
This is a mess. I've raised it here https://github.com/storybookjs/storybook/issues/11378#issuecomment-772375184 to see if there's a better way. 

Each story executes both in NodeJS and in a browser. I see no way to import the worker in a browser only.